charts/auth-layer-server/templates/realm-configmap.yaml

```diff
@@ -0,0 +1,2530 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: keycloak-realm-config
+data:
+  realm.json: |
+    {
+        "id": "84c3853a-a0a9-4a86-a15a-e8b850399200",
```

Comment:
Q: Are the ids in the file random and or configureable?

Also if anything is safely configureable it should be noted in the README or an additional config md file.
For a follow up PR if so

Reply:
Yes, all ids are randomly generated by KeyCloak when I created the Realm and configured it for the first time using the GUI and then Exported the config to a file for re-use. same am using as base on this helm chart.

The only thing that I am allowing to override or configure of this file is the `Client Secret`, which is defined on the `_helpers.tpl` as `auth.clientSecret` and basically looks for `.Values.global.auth.clientSecret` or `.Values.clientSecret` and if any of those exists it generates a new `random` value that later can be fetched from the UI.

That said, I agree that this should be noted on the README.md

Reply:
✅  Done